pg_search/src/postgres/customscan/pdbscan/parallel.rs

```diff
@@ -115,17 +121,3 @@ pub fn compute_nworkers(limit: Option<Cardinality>, segment_count: usize, sorted
 
     nworkers
```

Comment:
Mm: this method will need an update to use `min(num_cores, num_segments)` probably.